src/pages/previous.js

```diff
@@ -0,0 +1,152 @@
+import NavBar from "../components/navBar";
+import Chevron from "../assets/chevron_down.svg";
+import { GET } from "../utils/fetch";
+import { useEffect, useState } from "react";
+
+async function getQuestions() {
+  const weekly_questions = await GET("weekly_questions");
+  const questions = [];
+  for (const value in weekly_questions) {
+    questions.push({
+      id: weekly_questions[value].question_id,
+      body: weekly_questions[value].question.body,
+      time: new Date(weekly_questions[value].week),
+    });
+  }
+  return questions;
+}
+
+const options = {
+  month: "long",
+};
+
+function Previous() {
+  const [questions, setQuestions] = useState([]);
+  const [answers, setAnswers] = useState({});
```

Comment:
You could have the answers come nested in the questions, since it seems like that's what you want anyway. I think it would clean up the code a bit.